### PR TITLE
Get e2e tests working in prow

### PR DESCRIPTION
**Pull request by nckturner**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:

/kind failing-test

**What this PR does / why we need it**:

Fixes failing e2e test.

* Create test run directory earlier to avoid failure
* Install e2e prereqs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/e2e/run.sh [hunk 1]
```diff
@@ -155,9 +149,9 @@ if [[ "${UP}" = "yes" ]]; then
 fi
 
 pushd ./tests/e2e
-${GINKGO_BIN} . -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" "" -- -kubeconfig="${KUBECONFIG}" -report-dir="${test_run}" -gce-zone="${ZONES%,*}" "${EXPANDED_TEST_EXTRA_FLAGS}"
+ginkgo . -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" "" -- -kubeconfig="${KUBECONFIG}" -report-dir="${test_run}" -gce-zone="${ZONES%,*}" "${EXPANDED_TEST_EXTRA_FLAGS}"
 popd
 
 if [[ "${DOWN}" = "yes" ]]; then
-    kops delete cluster --name "${CLUSTER_NAME}" --yes
+    ${test_run}/kops delete cluster --name "${CLUSTER_NAME}" --yes
```
Comment (wongma7) [inline hack/e2e/run.sh:156]: how did the binary get here? is kops also an e2e-tool?

Comment (nckturner) [reply hack/e2e/run.sh:156]: Good question, its downloaded by the kubetest2 kops deployer: https://github.com/kubernetes/kops/blob/master/tests/e2e/pkg/kops/download.go#L32-L55

Comment (wongma7) [reply hack/e2e/run.sh:156]: should we call kubetest2 --down (or whatever the right arguments are) then instead, so we don't have to care about that implementation detail?

**Pull request merged**